tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -116,6 +119,60 @@ def verify_email(self, email):
         """Handle user email exceptions."""
         pass
 
+    def _handle_user(self, email, sub, auth_options):
+        """Handle user."""
+        User = get_user_model()
+
+        # Check if a user with the same email already exists
+        user = User.objects.filter(email=email).first()
```

Comment:
technically user can have blank email, and we are setting username as email, so I think you will need to change this to: user = User.objects.filter(username=email).first()